src/ritika/assignments_1to20/Asgmt_09_Webtable_Ops.java

```diff
@@ -0,0 +1,151 @@
+package ritika.assignments_1to20;
+
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import ritika.base.PredefinedActions;
+
+/*
+Q1 : How many rows are there in Employee Basic Information Table.
+Q2 : print last name of the employee whose first name is 'Dhara'.
+Q3 - A: print count of columns in the employee Basic Information table.
+Q3 - B: print all column name of the employee Basic Information table.
+Q4:  write a generic function to get column index of any column.
+Q5 - A: print first row of Employee basic table
+Q5 - B: print all rows of Employee basic table
+Q6: print username of all employees.
+Q7: print first name and last name of all employees.
+Q8: find unique last name from table Employee Basic.
+Q9: how many employees (rows) are there after "Dhara".
+*/
+public class Asgmt_09_Webtable_Ops {
+
```

Comment:
You can define driver here as instance variable, so you don't require to pass driver reference to each method